### PR TITLE
Follow-up: Do not send logs to CI if the credentials are not set

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
This is an automated comment for commit 784698b9f62d65181d8c4ca05c21d2c66dba7b2c with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/53456/784698b9f62d65181d8c4ca05c21d2c66dba7b2c/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: tests/ci/ast_fuzzer_check.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def main():
         "CLICKHOUSE_CI_LOGS_PASSWORD", "CLICKHOUSE_CI_LOGS_PASSWORD"
     )
 
-    if ci_logs_host != "CLICKHOUSE_CI_LOGS_HOST":
+    if ci_logs_host != "CLICKHOUSE_CI_LOGS_HOST" and ci_logs_host != "":
```
Comment (tavplubix) [inline tests/ci/ast_fuzzer_check.py:149]:
```suggestion
    if ci_logs_host not in ('CLICKHOUSE_CI_LOGS_HOST', ''):
```

File: tests/ci/functional_test_check.py [hunk 2]
```diff
@@ -394,7 +394,7 @@ def main():
     ci_logs_password = os.getenv(
         "CLICKHOUSE_CI_LOGS_PASSWORD", "CLICKHOUSE_CI_LOGS_PASSWORD"
     )
-    if ci_logs_host != "CLICKHOUSE_CI_LOGS_HOST":
+    if ci_logs_host != "CLICKHOUSE_CI_LOGS_HOST" and ci_logs_host != "":
```
Comment (tavplubix) [inline tests/ci/functional_test_check.py:397]:
```suggestion
    if ci_logs_host not in ('CLICKHOUSE_CI_LOGS_HOST', ''):
```

File: tests/ci/stress_check.py [hunk 3]
```diff
@@ -209,7 +209,7 @@ def run_stress_test(docker_image_name):
     ci_logs_password = os.getenv(
         "CLICKHOUSE_CI_LOGS_PASSWORD", "CLICKHOUSE_CI_LOGS_PASSWORD"
     )
-    if ci_logs_host != "CLICKHOUSE_CI_LOGS_HOST":
+    if ci_logs_host != "CLICKHOUSE_CI_LOGS_HOST" and ci_logs_host != "":
```
Comment (tavplubix) [inline tests/ci/stress_check.py:212]:
```suggestion
    if ci_logs_host not in ('CLICKHOUSE_CI_LOGS_HOST', ''):
```

**Review by tavplubix — APPROVED** [review]

**Comment by tavplubix** [comment]
That's funny that `pylint` suggests
`/ClickHouse/tests/ci/ast_fuzzer_check.py:149:7: R1714: Consider merging these comparisons with "in" to "ci_logs_host not in ('CLICKHOUSE_CI_LOGS_HOST', '')" (consider-using-in)`
but then `black` replaces single quotes with double quotes

**Comment by azat** [comment]
Do I understand correctly that right now sending logs to CI had been disabled (I guess secret vars are not set?)?

And this PR is required to unbreak the CI:

```
subprocess.CalledProcessError: Command 'sed -i -r -e 's!!CLICKHOUSE_CI_LOGS_HOST!g; s!!CLICKHOUSE_CI_LOGS_PASSWORD!g;' '/home/ubuntu/actions-runner/_work/_temp/stateless_database_replicated/result_path/run.log'' returned non-zero exit status 1.
```

**Comment by tavplubix** [comment]
> Do I understand correctly that right now sending logs to CI had been disabled

No

> I guess secret vars are not set?

Yes, they are not set for PRs from forks

> And this PR is required to unbreak the CI

Yes

**Pull request merged**